scripts/dev.js

```diff
@@ -34,7 +34,13 @@ const owWaitPeriodTime = 500
 const owTimeout = 60000
 
 class ActionServer extends BaseScript {
-  async run (args = [], bundleOptions = {}) {
+  async run (args = [], options = {}) {
```

Comment:
I think a lot of these changes are in your other pr ...

Reply:
Yes I  know I will make sure to merge things properly, as #102 builds on top of this one. The reason is that both PRs touch dev.test.js which is fragile, and I didn't want to have additional merge problems